include/zephyr/net/dns_resolve.h

```diff
@@ -175,10 +174,10 @@ struct dns_resolve_context {
 		struct net_context *net_ctx;
 
 		/** Is this server mDNS one */
-		uint8_t is_mdns : 1;
+		uint8_t is_mdns: 1;
 
 		/** Is this server LLMNR one */
-		uint8_t is_llmnr : 1;
+		uint8_t is_llmnr: 1;
```

Comment:
and here too. In fact it seems that all the changes to `dns_resolve.h` are cosmetic and should be removed from this PR.